WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2596,10 +2549,6 @@ title_master_of_the_royal_elephants = {
 	}
 	
 	allowed_to_grant = {
-		OR = {
-			is_feudal = yes
-			is_tribal = yes
-		}
```

Comment:
@loup99 

Are you saying that an Indian cultured character ceases to use elephants as a republic? That's crazy. The conditions should not exist.